runners/java-fn-execution/src/main/java/org/apache/beam/runners/fnexecution/control/DefaultJobBundleFactory.java

```diff
@@ -464,6 +474,8 @@ ServerInfo getServerInfo() {
     }
 
     public void close() {
```

Comment:
Isn't close only called from unref? If so, how does this change the behavior? (Possibly some more explanation needs to be added.)

Reply:
It's now also called from here: https://github.com/apache/beam/pull/10694/files#diff-e80c769f0011537cc2b60d3e7898cf5aR260